shopfloor_ui_tests/cypress/fixtures/app_config_test.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
what's the difference w/ the other file beside the user name? Shall we have better file naming?